pkg/kubectl/cmd/rollingupdate.go

```diff
@@ -298,10 +305,14 @@ func RunRollingUpdate(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, arg
 	}
 
 	if keepOldName {
-		fmt.Fprintf(out, "%s\n", oldName)
-	} else {
-		fmt.Fprintf(out, "%s\n", newRc.Name)
+		newRc.Name = oldName
+	}
+	newRc, err = client.ReplicationControllers(cmdNamespace).Get(newRc.Name)
+	if outputFormat != "" {
+		return f.PrintObject(cmd, newRc, out)
 	}
+	_, resource := meta.KindToResource(newRc.Kind, false)
+	cmdutil.PrintSuccess(mapper, false, out, resource, oldName, "rolling updated to \""+newRc.Name+"\"")
```

Comment:
if `keepOldName`, should we print `<resource> "A" rolling updated to "A"` or just `<resource> "A" rolling updated`?